modules/secrets-store-providers.adoc

```diff
@@ -6,10 +6,15 @@
 [id="secrets-store-providers_{context}"]
 = Secrets store providers
 
-The following secrets store providers are available for use with the {secrets-store-operator}:
+The {secrets-store-operator} has been tested with the following secrets store providers:
 
 * AWS Secrets Manager
 * AWS Systems Manager Parameter Store
 * Azure Key Vault
 * Google Secret Manager
 * HashiCorp Vault
+
+[NOTE]
+====
+Red{nbsp}Hat does not test all factors associated with third-party secrets store provider functionality. For more information about third-party support, see the link:https://access.redhat.com/third-party-software-support[Red{nbsp}Hat third-party support policy].
+====
```

Comment:
qq - does it make sense to link the SSCSI acceptance requirements from the upstream? 
https://secrets-store-csi-driver.sigs.k8s.io/providers#criteria-for-supported-providers

Reply:
Although we can link to upstream sometimes, we do try to avoid it unless it's truly needed. In this case I'm not sure that it is, since we are listing specific supported providers, not just any provider that meets these requirements.

Reply:
👍 makes sense. Thanks!